code/script.js

```diff
@@ -1,325 +1,380 @@
-// All the DOM selectors stored as short variables
-const board = document.getElementById('board')
-const questions = document.getElementById('questions')
-const restartButton = document.getElementById('restart')
+//DOM selectors declared here
+const board = document.getElementById("board");
+const dropdown = document.getElementById("dropdown");
+const playAgainButton = document.getElementById("playAgain");
+const restartButton = document.getElementById("restartBtn");
+const filterButton = document.getElementById("filter");
+const questionCounter = document.getElementById("questions-left");
 
-// Array with all the characters, as objects
+//Array with all the characters as objects
 const CHARACTERS = [
   {
-    name: 'Jabala',
-    img: 'images/jabala.svg',
-    hair: 'hidden',
-    eyes: 'hidden',
-    accessories: ['glasses', 'hat'],
-    other: []
+    name: "Biscuit",
+    img: "images/Biscuit.png",
+    color: "dotted",
+    job: "designer",
+    accessories: ["hat", "jewelry"],
+    other: [],
   },
   {
```

Comment:
Theme-wise, what can I say other than it's an absolute heart-stealer. The .png's matching the characters' names is a neat approach for the code's readability as well.